assets/src/styles/blocks/Socialshare.scss

```diff
@@ -0,0 +1,86 @@
+.social-share-block-wrap {
```

Comment:
This files has multiple linting errors. You check them locally by running:

 `./node_modules/stylelint/bin/stylelint.js assets/src/styles/blocks/Socialshare.scss`